src/labels_by_user_input.py

```diff
@@ -10,25 +12,60 @@
 )
 
 
+LABELS_SET = {
+    "hold": "do-not-merge/on-hold",
+    "wip": "do-not-review/wip",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+# Newline characters aren't supported in f-strings
+UNSUPPORTED_LABELS = """
+You're trying to add/remove an unsupported label.
+Supported labels -
+{}
```

Comment:
`{}` use fstring

Reply:
I'm seperating labels with newlines, which aren't supported by fstring

Reply:
you can assign `\n` to a variable and use it within f-string